.github/actions/update-dep-version/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM alpine
+
+RUN apk update && \
+    apk upgrade && \
+    apk add git && \
+    apk add go && \
+    apk add jq && \
+    apk add nodejs npm && \
+    apk add make && \
+    git clone https://github.com/cli/cli.git gh-cli && \
+    cd gh-cli && \
+    make && \
+    mv ./bin/gh /usr/local/bin/
```

Comment:
@akandeBolaji remove cache and repo. Also, some build tools (make) are not needed.

Reply:
@rkalkani `make` is needed to build `./bin/gh`. Can you also be more specific with removing cache and repo?